### DISCUSSION TITLE
Public money, public code

### DISCUSSION THREAD

**Discussion by NikosAlexandris** [OSGeo/grass#1925] [General]
Post by @ninsbl in [OSGeo Discuss mailing list](Discuss@lists.osgeo.org):

> Hi,
> 
> Not sure how well known the “Public money, public code” campaign of the Free Software Foundation Europe (fsfe) is within the circles of OSGeo:
> 
> https://publiccode.eu/
> 
> FOSSGIS e.V. and GFOSS signed it and I have probably overlooked other OSGeo related organizations. But maybe it is relevant for OSGeo itself as well?
> 
> Though it has a European focus, the information material there is very good can be useful also in other parts of the world, when arguments for using Free and Open Source Software in or for public authorities are needed… 
> 
> Check it out.
> 
> Cheers
> Stefan

**Comment by lucadelu** [comment]
For my side it make sense if organization (like GFOSS, FOSSGIS and OSGeo) sign it, not so single project like GRASS
